drivers/driver.go

```diff
@@ -244,13 +270,16 @@ func (r RDMAVolumeDriver) Unmount(request volume.UnmountRequest) volume.Response
 
 	r.validateOrCrash()
 
+	// Pass the unmount request to the volume database.
+	err := r.VolumeDatabase.Unmount(request.Name, request.ID)
+
 	// Pass the unmount request to the storage controller.
-	err := r.StorageController.Unmount(request.Name, request.ID)
+	// TODO err := r.StorageController.Unmount(request.Name, request.ID)
```

Comment:
TODOs should be made into issues and linked to the issue in the code comment with at least a #number. Otherwise, they should be removed.